components/raftstore-v2/src/operation/ready/snapshot.rs

```diff
@@ -217,9 +217,18 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
                 }
             }
             SnapState::Generated(ref s) => {
+                if s.1 != to {
+                    info!(self.logger(),
+                        "need previous peer to call the snapshot";
```

Comment:
What if the previous peer doesn't need snapshot anymore?

Reply:
It will cancel the snapshot if the previous peer doesn't the snapshot  anymore, so the `SnapState` will become Relax.

Reply:
At which step?